data/publiccloud/terraform/sap/azure.tfvars

```diff
@@ -1,42 +1,79 @@
-# Launch SLES-HAE of SLES4SAP cluster nodes
+# No bastion connection
+bastion_enabled = false
 
-# Instance type to use for the cluster nodes
-hana_vm_size = "%MACHINE_TYPE%"
+# Enable all some pre deployment steps (disabled by default)
+pre_deployment = true
 
-# Disk type for HANA
-hana_data_disk_type = "Premium_LRS"
+# Region where to deploy the configuration
+az_region = "%REGION%"
```

Comment:
so basically you just move variable from one place to another in a file without changing it . is it really needed ? I just thinking about commit history and how it will look like . It would be not so obvious after some time to figure out that this value haven't changed

Reply:
Same answer as previous comment: I "simply" modified the default configuration file, so some things may have been swapped.

Reply:
ok